src/hooks/useCommitmentFilter.js

```diff
@@ -25,12 +25,16 @@ const useCommitmentFilter = () => {
     });
   }
 
-  function isPlanned(commitment) {
-    let planned = false;
-    if (commitment.confirm_by > Math.floor(Date.now() / 1000)) {
-      planned = true;
-    }
-    return planned;
+  function getCommitmentLabel(commitment) {
+    let label;
+    isActive(commitment)
+      ? (label = "Committed")
+      : isPending(commitment)
+        ? (label = "Pending")
+        : isPlanned(commitment)
+          ? (label = "Planned")
+          : (label = "");
+    return label;
```

Comment:
I see what you're going for, but I was initially confused because it looked like `label = "value"` is a check, which it isn't. This would be more readable as a basic sequence of ifs:

```suggestion
    if (isActive(commitment)) { return "Committed"; }
    if (isPending(commitment)) { return "Pending"; }
    if (isPlanned(commitment)) { return "Planned"; }
    return "";
```